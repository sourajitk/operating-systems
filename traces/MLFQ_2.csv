1,0,10
2,3,3
3,4,1
4,6,3
5,9,2
6,10,8