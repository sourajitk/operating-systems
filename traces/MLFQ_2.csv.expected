1,27
2,9
3,5
4,14
5,12
6,25